Translations/UltraPermissions_Russian.lang

```diff
@@ -1,171 +1,172 @@
 OVERVIEW: Обзор
-PERMISSION: Право
-POSITIVE: Положительный
-NEGATIVE: Отрицательный
-MIXED: Смешанный
-BUNGEE_PERMISSION: Права Bungee
+PERMISSION: право
+POSITIVE: Положительн.
```

Comment:
Why is there a dot at the end? Do they have any special meaning in Russian? I want to make sure since it's only one word.

Reply:
Because in the plugin, this word is inserted in several lines with different cases, and without abbreviations it is read incorrectly!

This is because there are 6 cases in Russian!

Reply:
Example: 
<img width="660" alt="CleanShot 2020-06-02 at 17 18 19@2x" src="https://user-images.githubusercontent.com/7906418/83519257-15542400-a4f5-11ea-98b9-a700b10a89ee.png">

In title it is should "ПоложительнОЕ право" but in description it is like an action "Сделать право положительнЫМ"

Reply:
Same with negative and mixed